index.bs

```diff
@@ -6024,8 +6021,6 @@ const graph = await builder.build({'output': output});
         <td>{{Uint8Array}}
 </table>
 
-<p class="note">{{Float16Array}} is at <a href="https://tc39.es/process-document/">ECMA Stage 3</a> signaling its design is finished. Implementers wanting to enable this type ahead native implementations can emulate the type by passing raw bits via {{Uint16Array}}. <a href="https://github.com/webmachinelearning/webnn/issues/373">[Issue webnn#373]</a></p>
```

Comment:
Is it live in any implementations yet though? Should we keep the "implementations can emulate the type" hint until then?

Reply:
Reasonable - I thought it was already going out but I'm not finding the target milestone in Chrome. I also wasn't sure who "Implementers" refers to in that sentence. I'll put that back in for now.

Reply:
Back in 18fae72cc4efc86ae7d18959dba0b8e5c1899aa5